analytics/migrations/0004_add_subgroup.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
```

Comment:
These should be split into two migrations, one for the subgroup, and another for the AlterFields.